tts-service/app/tts_manager.py

```
@@ -451,10 +425,39 @@ async def generate_speech(
+                        )
+                        async with self.model_load_lock:
+                            self.use_cuda = False
+                            self.device = "cpu"
+                            self.loaded_models.clear()
+                            self.synthesizer = None
+                            self.current_model = None
+                            if torch.cuda.is_available():
+                                torch.cuda.empty_cache()
+                            tts = await self._load_model_impl(model_name)
+                        if not tts:
+                            raise ValueError(f"Failed to load model {model_name} on CPU for fallback") from e
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)